tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,99 @@
+"""Simple demo which displays constant 3D mesh."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import numpy as np
+import tensorflow as tf
+import trimesh
+
+from tensorboard.plugins.mesh import summary as mesh_summary
+
+tf.flags.DEFINE_string('logdir', '/tmp/mesh_demo',
+                       'Directory to write event logs to.')
+tf.flags.DEFINE_string('sample_meshes', '',
+                       'Path to folder with PLY files or string of comma '
+                       'separated PLY files to visualize.')
+tf.flags.DEFINE_integer('batch_size', 1,
+                        'Size of the batch. All provided PLY files will be '
+                        'divided into batch_size groups and used at different '
+                        'steps during training.')
+FLAGS = tf.flags.FLAGS
+
+
```

Comment:
here, I'd add `tf.compat.v1.disable_v2_behavior()` for interop with TensorFlow 2.0 preview